root/static/scripts/release-editor/fields.js

```diff
@@ -294,6 +295,10 @@ class Track {
         return hasVariousArtists(this.artistCredit());
     }
 
+    hasFeatOnTitle() {
+        return this.name().match(featRegex);
```

Comment:
`.test()` is usually preferred if we don't use the match and just want a boolean
```suggestion
        return featRegex.test(this.name());
```